geopandas/tests/test_cmapplotting.py

```diff
@@ -0,0 +1,240 @@
+from __future__ import absolute_import, division
+
+import itertools
+import warnings
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+from shapely.geometry import Point, Polygon, LineString
+
+from geopandas import GeoSeries, GeoDataFrame, plotting
+from geopandas.tests.test_plotting import _check_colors
+
+import pytest
+
+@pytest.fixture(autouse=True)
+def close_figures(request):
+    yield
+    plt.close('all')
+
+try:
+    cycle = mpl.rcParams['axes.prop_cycle'].by_key()
+    MPL_DFT_COLOR = cycle['color'][0]
+except KeyError:
+    MPL_DFT_COLOR = matplotlib.rcParams['axes.color_cycle'][0]
+
+class TestPointPlotting:
+    def setup_method(self):
+        # Intializing the dataframe
+        self.N = 10
+        self.points = GeoSeries(Point(i, i, i) for i in range(self.N))
+
+        values = np.arange(self.N)
+        levels = ['1','1','1','1','1','1','2','2','2','2']
```

Comment:
Can you make this numbers instead of strings?

Reply:
Also, I don't think you are using this in this class, so I would remove it here.

Reply:
The points are not giving the expected result while using the subset, so I only used 2 tests for testing Points.